tests/operators/test_python.py

```diff
@@ -817,6 +822,31 @@ def f():
 
         self._run_as_operator(f, requirements=['funcsigs', 'dill'], system_site_packages=False)
 
+    def test_private_repo_requirements(self):
+
+        with create_session() as session:
+            self._clean_stop_pypi_server()
+            self._initiate_pypi_server()
```

Comment:
This could be pytest fixture or at the very least it should be either in try/finally or in setUp()/tearDown(). otherwise there is a risk the server stays running after test throws an exception.